.gitignore

```diff
@@ -1,4 +1,3 @@
 /target
 *.iml
 /.idea
-xenon.yml
```

Comment:
This was intentional to have `xenon.yml` in the .gitignore - this is so you can have your own personal config and not worry about accidentally committing it when making changes to Xenon.